src/core/onResize.js

```diff
@@ -1,5 +1,7 @@
 import refresh from "./refresh";
 
 export default function onResize() {
-  refresh.call(this);
+  if (document.querySelector('.introjs-tooltip') !== null) {
```

Comment:
I think it makes more sense to move this if-else to the `refresh` function.

Reply:
I'm pretty sure this would fix https://github.com/usablica/intro.js/issues/1282 (as well as https://github.com/HiDeoo/intro.js-react/issues/70)

Reply:
> I'm pretty sure this would fix #1282 (as well as [HiDeoo/intro.js-react#70](https://github.com/HiDeoo/intro.js-react/issues/70))

After some more digging, I believe these are separate error paths. The bug in the ticket I referenced is triggered when there is no element on the current step (https://github.com/usablica/intro.js/blob/master/src/core/refresh.js#L43), whereas this PR would correct for a missing `.introjs-tooltip`, which is the next parameter in `placeTooltip` (https://github.com/usablica/intro.js/blob/master/src/core/refresh.js#L44).